src/files.js

```diff
@@ -307,8 +307,12 @@ export function restoreById (cozy, id) {
   return cozyFetchJSON(cozy, 'POST', `/files/trash/${encodeURIComponent(id)}`)
 }
 
-export function destroyById (cozy, id) {
-  return cozyFetchJSON(cozy, 'DELETE', `/files/trash/${encodeURIComponent(id)}`)
+export function destroyById (cozy, id, opts) {
+  return cozyFetchJSON(cozy, 'DELETE', `/files/trash/${encodeURIComponent(id)}`, undefined, {
+    headers: {
+      'If-Match': (opts && opts.ifMatch) ? opts.ifMatch : ''
```

Comment:
Finally, I copied how it was done in other functions with an intermediary ifMatch variable.